ckanext/showcase/commands/migrate.py

```diff
@@ -17,11 +17,22 @@ class MigrationCommand(CkanCommand):
         paster showcase migrate -c <path to config file>
             - Migrate Related Items to Showcases
 
+        paster showcase migrate -c <path to config file> [--allow-duplicates]
+            - Migrate Related Items to Showcases and allow duplicates
+
     Must be run from the ckanext-showcase directory.
     '''
     summary = __doc__.split('\n')[0]
     usage = __doc__
 
+    def __init__(self,name):
+        super(CkanCommand, self).__init__(name)
+
+        self.parser.add_option('--allow-duplicates', dest='allow_duplicates',
+                            default=False, help='''Use this option to allow duplicate relations
+                            to be migrated. Showcases will be created as
+                            'duplicate_related-name_package-name'.''', action='store_true')
```

Comment:
Two small things about the help text:

1) Only subsequent duplicate showcases will be created with the title 'duplicate_...'. The first migrated showcase will have the same title as the related item
2) The name format is now `'duplicate_<related-name>_<related-id>'`

Suggest: 

`"Use this option to allow related items with duplicate titles to be migrated. Duplicate showcases will be created as 'duplicate_<related-name>_<related-id>'."`